### PR TITLE
feat: add basic dark theme

### PR DESCRIPTION
**Pull request by Rodney1988**
added a few html element theme colors based on a toggle button, you can choose the exact changes. Let me know if all is good.

### CODE REVIEW

File: src/components/DarkModeToggle.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      localStorage.setItem("theme", "light");
+    } else {
+      document.documentElement.classList.add("dark");
+      localStorage.setItem("theme", "dark");
+    }
+    setIsDarkMode(!isDarkMode);
+  };
+
+  return (
+    <button
```
Comment (Bashamega) [inline src/components/DarkModeToggle.jsx:27]: In stead of a button, how about we add a dark mode toggle?

**Review by Bashamega — CHANGES_REQUESTED** [review]

**Comment by Rodney1988** [comment]
hey @Bashamega, I changed it to a toggle button.

**Review by Bashamega — CHANGES_REQUESTED** [review]
Great work!

Please run `npm run prettier:write` to fix the failing checks

**Comment by Rodney1988** [comment]
there you go :)

**Review by Bashamega — APPROVED** [review]

**Comment by Bashamega** [comment]
Great work:)

**Pull request merged**